notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -336,13 +336,14 @@
       "source": [
```

Comment:
<p>move imports to the single import cell earlier.</p><p> </p><p>Endpoint and Predict are supported in SDK, why are you using GAPIC?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Imports fixed.</p><p> </p><p>Why GAPIC? Because Soheila agreed to do the update to the SDK so she's doing that as her first work item after this PR is merged.</p>

Reply:
got it